.github/workflows/ci.yml

```diff
@@ -9,3 +9,6 @@ jobs:
   plugin-ci:
     uses: mattermost/actions-workflows/.github/workflows/plugin-ci.yml@f53a9d32cc670c0319d95332e07c68c00b332928
     secrets: inherit
+    with:
+      golang-version: 1.24
```

Comment:
We are on Go 1.23.7

Reply:
You are in v1.22.8 https://github.com/mattermost/mattermost-plugin-confluence/blob/0d2815f6b66898ae946c911610cd1294707f73ac/go.mod#L5

But inside the PR #140, related to the update of `github.com/mattermost/mattermost/server/public` you will be in v1.23

https://github.com/mattermost/mattermost/blob/34fb9adbc25157de8cb2a77d95aaa9fd9b97830d/server/go.mod#L5